bundles/org.openhab.binding.velux/README.md

```diff
@@ -110,12 +110,19 @@ Notes:
 1. To enable a complete inversion of all parameter values (i.e. for Velux windows), use the property `inverted` or add a trailing star to the eight-byte serial number.
```

Comment:
```suggestion
1. To enable a complete inversion of all parameter values, use the property `inverted` or add a trailing star to the eight-byte serial number.
```
I would suggest to drop `(i.e. for Velux windows)` as windows don't need to be inverted to work as expected.